index.html

```diff
@@ -0,0 +1,386 @@
+                  src="images/Counter.png" alt="counter1"></span>
+              <span class="inrow1"> 2018</span>
+            </p><br>
+            <p class="innerpara1">A smart assistant to make driving more safe, efficient, and fun by unlocking your
+              most expensive computer: your car.
+            </p><br>
+            <ul>
+              <li class="back4">html</li>
+              <li class="back4">Ruby on rails</li>
+              <li class="back4">css</li>
+              <li class="back4">javascript</li>
+            </ul><br>
+            <button type="button" class="button button4">See Project</button>
+          </div>
+          <div class="workimage1">
+            <img src="images/Snapshoot-Portfolio4.png" alt="3rd work immage">
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="workimage">
+            <img src="images/Snapshoot3.png" alt="3rd work immage">
+            <h2 class="tonic">Multi-Post</h2>
+            <h2 class="tonic"> Stories</h2> <br>
+            <p>
+              <span class="inrow"><b>CANOPY</b> </span><span class="inrow"><img src="images/Counter.png"
+                  alt="counter"></span><span class="inrow"> Back End Dev </span><span class="inrow"><img
+                  src="images/Counter.png" alt="counter"></span>
+              <span class="inrow"> 2015</span>
+            </p><br>
+            <p class="innerpara">A daily selection of privately personalized reads; no accounts or sign-ups
+              required.</p><br>
+            <ul>
+              <li class="back">html</li>
+              <li class="back">css</li>
+              <li class="back">javascript</li>
+            </ul><br>
+            <button type="button" class="button button4">See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+  </div>
+
+  <div class="container">
+    <section class="back2">
+      <div class="item1">
+
+        <p class="bold2">About<span><br>Myself</span></p>
+        <p class="customise customise1">Hello I’m a software developer! I can help you build a product , feature or website Look
+          through some of my work and experience! If you like what you see and have a project you need coded, don’t
+          hestiate
+          to contact me.</p>
+        <h3 class="letsconnect">LET'S CONNECT</h3>
+        <div class="social-media">
+          <ul class="onerow1">
+            <li><a href="https://twitter.com/"><i class="fa-brands fa-twitter"></i></a></li>
+            <li><a href="https://www.linkedin.com/"><i class="fa-brands fa-linkedin-in"></i></a></li>
+            <li><a href="https://medium.com/"><i class="fa-brands fa-medium"></i></a></li>
+            <li><a href="https://github.com/"><i class="fa-brands fa-github"></i></a></li>
+            <li><a href="https://angel.co/"><i class="fa-brands fa-angellist"></i></a></li>
+          </ul>
```

Comment:
- [ ] Kindly match your social media icons like the one in Figma, you can extract the icons and use them in your design.

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/173161099-2ef807f8-e823-4214-8e0b-1ca282a252fc.png)| ![image](https://user-images.githubusercontent.com/53402600/173161169-a6f59cc0-4864-4852-a89d-58bd713ea378.png) |